partitionmanager/cli.py

```diff
@@ -296,9 +296,15 @@ def do_partition(conf):
         help_text="Time in seconds to complete the ALTER command",
         type_name="gauge",
     )
+    metrics.describe(
+        "alter_errors",
+        help_text="Number of errors observed during ALTER commands",
+        type_name="counter",
+    )
 
     all_results = dict()
     for table in conf.tables:
+        time_start = None
```

Comment:
I was confused when reviewing but I see we set time_start to now on line 345; this is just to pull it into a higher scope